rosdep/python.yaml

```diff
@@ -4082,15 +4114,17 @@ python3-flake8:
   ubuntu: [python3-flake8]
 python3-gitlab:
   debian:
-    '*': [python3-gitlab]
-    jessie:
+    '*':
```

Comment:
the wildcard should match the apt package as this will be the default used for any new packages. So widlcards should not resolve to pip rules.